documentation/projects/proposals/rekognition_data/20240320-project_proposal_rekognition_data.md

```diff
@@ -0,0 +1,221 @@
+Rekognition are available in both the API and the frontend.
+
+If the labels themselves are observed to have a negative impact on search
+relevancy, we will need a mechanism or plan for the API for suppressing or
+deboosting the machine-labeled tags without having to remove them entirely (_NB:
+We may be able to leverage some of the DAGs created as a part of the
+[search relevancy sandbox](../search_relevancy_sandbox/20230331-project_proposal_search_relevancy_sandbox.md)
+project for this rollback_).
+
+## Participants and stakeholders
+
+<!-- Who is working on the project and who are the external stakeholders, if any? Consider the lead, implementers, designers, and other stakeholders who have a say in how the project goes. -->
+
+- **Lead**: @AetherUnbound
+- **Design**: @fcoveram _(if any frontend design is deemed necessary)_
+- **Implementation**: Implementation may be necessary for the frontend, API, and
+  catalog; all developers working on those aspects of the project could be
+  involved.
+
+## Infrastructure
+
+<!-- What infrastructural considerations need to be made for this project? If there are none, say so explicitly rather than deleting the section. -->
+
+The Rekognition data presently exists in an S3 bucket that was previously
+accessible to @zackkrida. We will need to ensure that the bucket is accessible
+by whatever resources are chosen to process the data. This was
+[previously done](https://github.com/WordPress/openverse/issues/431#issuecomment-1675434911)
+by manually instantiating an EC2 instance to run
+[a python script which generated a labels CSV](https://gist.github.com/zackkrida/cb125155e87aa1c296887e5c27ea33ff).
+We may instead wish to either run any pre-processing locally or set up an
+Airflow DAG which would perform the processing for us.
+
+## Accessibility
+
+<!-- Are there specific accessibility concerns relevant to this project? Do you expect new UI elements that would need particular care to ensure they're implemented in an accessible way? Consider also low-spec device and slow internet accessibility, if relevant. -->
+
+The greatest concern on accessibility would be ensuring whatever mechanism we
+use for conveying the machine-generated nature/accuracy values in the frontend
+is also reflected in a suitable manner for screen readers.
+
+## Marketing
+
+<!-- Are there potential marketing opportunities that we'd need to coordinate with the community to accomplish? If there are none, say so explicitly rather than deleting the section. -->
+
+We should share the addition of the new machine-generated tags publicly once
+they are present in both the API and the frontend.
+
+## Required implementation plans
+
+<!-- What are the required implementation plans? Consider if they should be split per level of the stack or per feature. -->
+
+The requisite implementation plans reflect the primary pieces of the project
+described above:
+
+- Determine and design how machine-generated tags will be displayed/conveyed in
```

Comment:
Should there be an additional IP for determining accuracy cut-offs and which tags will actually be used? If not, which of these IPs will that work be part of?